editor/modes/visual-editor/index.js

```diff
@@ -40,6 +52,20 @@ class VisualEditor extends Component {
 		}
 	}
 
+	onKeyDown( event ) {
+		const { uids } = this.props;
+		const isEditable = [ 'textarea', 'input', 'select' ].indexOf( document.activeElement.tagName.toLowerCase() ) !== -1
```

Comment:
The idea was to enable Ctrl+A even if we're focusing a button or any focusable but not editable. We could restrict to document.body of course.